headers/types/common/enums.h

```diff
@@ -3,6 +3,12 @@
 #ifndef HEADERS_TYPES_COMMON_ENUMS_H_
 #define HEADERS_TYPES_COMMON_ENUMS_H_
 
+// Represents one of the two screens the console has
```

Comment:
What does this mean? Like the top screen and the touch screen? Can we use more descriptive names for the enum labels? It's not clear to me what "main" means in this context.